### ISSUE TITLE
Canvas to draw on

### ISSUE THREAD

**Issue by fonsp** [fonsp/PlutoUI.jl#9]

**Comment by RensDimmendaal** [comment]
I'm excited about this feature. I augmented the box drawing feature for a small demo: https://twitter.com/R_Dimm/status/1305251396669800448

I don't know javascript but potentially this might serve as inspiration: https://github.com/andfanilo/streamlit-drawable-canvas they had a similar usecase but in python.

I'd be happy to take a stab at this feature someone can help specify the requirements.

**Comment by fonsp** [comment]
Sorry I already kind of wrote this myself:
https://github.com/fonsp/disorganised-mess/blob/master/drawoncanvas.jl

**Comment by RensDimmendaal** [comment]
Super. Keep up the great work!

**Comment by shashankp** [comment]
Great to see already something in the works..
I was prototyping about a **2D slider** 

![image](https://user-images.githubusercontent.com/498813/93135233-a2636780-f6f7-11ea-906b-17ff166758f2.png)

**Comment by fonsp** [comment]
@dpsanders asked for that once

**Comment by shashankp** [comment]
Create another issue to track it?

**Comment by fonsp** [comment]
yep!

**Comment by DhruvaSambrani** [comment]
> Sorry I already kind of wrote this myself: https://github.com/fonsp/disorganised-mess/blob/master/drawoncanvas.jl
> 
> slightly_smiling_face let's turn it into a PlutoUI element after #3 is done

Hi @fonsp what's the scene on this issue? Is it just a matter of someone formalizing your code?

**Comment by fonsp** [comment]
WIP at #234!
